### DISCUSSION TITLE
当产生标注时，如何同时下载标注与原文

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2147] [服务]
# 注意

> ### 嗨，简悦全新的 [教程中心](https://www.yuque.com/kenshin/simpread) 已上线，**下面的内容并不一定是最新的**，建议 [访问这里](https://www.yuque.com/kenshin/simpread/wkswh7) 查看最新版教程。

**Comment by xleonzhao** [comment]
请问这个问题到底怎么解决的呢？好像两个文件用的是同一个文件名，最后存的只有标注

**Reply by Kenshin** [reply]
你需要配置本地快照 https://www.yuque.com/kenshin/simpread/wkswh7

再有问题请通过 https://github.com/kenshin/simpread/issues/new 讨论
